voxel51/users/query.py

```diff
@@ -97,6 +97,18 @@ def add_search(self, field, search_str):
             self.search.append("%s:%s" % (field, search_str))
         return self
 
+    def add_search_or(self, field, search_strs):
+        '''Adds the given "OR" search to the query.
```

Comment:
recommend adding a sentence or two here and in `add_search` explaining that `add_search` matches any records whose `field` contains the given search string, and `add_search_or` matches any records whose `field` contains any of the given search strings